zkevm-circuits/src/evm_circuit/execution/callop.rs

```diff
@@ -271,6 +265,10 @@ impl<F: Field> ExecutionGadget<F> for CallOpGadget<F> {
                 cb.power_of_randomness(),
             ),
         );
+        // is_empty_code_hash_expr is true when the account exists and has empty
+        // code hash, or when the account doesn't exist (which we encode with
+        // code_hash = 0).
+        let is_empty_code_hash_expr = is_empty_code_hash.expr() + callee_not_exists.expr();
```

Comment:
I think a better name would be `no_callee_code`. `is_empty_code_hash_expr` looks like it should be just `is_empty_code_hash.expr()`.

Reply:
Thanks for the suggestion!  Applied in https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1046/commits/8c84bc31ed1ffc74c31508b5818e90d8762dd450